common/install_android.sh

```diff
@@ -104,3 +104,83 @@ export PATH="${GRADLE_HOME}/bin/:${PATH}"
 echo "PATH:${PATH}"
 
 gradle --version
+
+rm -rf /tmp/gradledeps
+
+_gradledeps=/tmp/gradledeps
+
+rm -rf $_gradledeps
+
+cat <<EOF >"$_gradledeps/AndroidManifest.xml"
+<manifest package="org.pytorch" />
+EOF
+
+cat <<EOF >"$_gradledeps/build.gradle"
```

Comment:
Can this be a separate checked-in file instead of a heredoc?

Reply:
It's possible if change /ubuntu/Dockerfile that has:
```
ADD ./common/install_android.sh install_android.sh
RUN if [ -n "${ANDROID}" ]; then bash ./install_android.sh; fi
RUN rm install_android.sh
```
and the same for other docker images.

```
ADD ./android/build.gradle build.gradle
ADD ./android/AndroidManifest.xml AndroidManifest.xml
...
```
That will be a bit different with other images, but in separate files